.changelog/README.md

```diff
@@ -0,0 +1,132 @@
+# How To Use
+
+Nomad uses [`go-changelog`](https://github.com/hashicorp/go-changelog) to generate its changelog on release.
+
+To install, run the following command:
+
+```
+go get github.com/hashicorp/go-changelog/cmd/changelog-build
+```
+
+## Developer Guide
+
+PRs with code changes should include a `.changelog/<PR Number>.txt` file highlighting the user visible change.
+
+The changelog entry should include the affected component (e.g. `driver/docker`, `api`), and a user-centric summary of the change. If an issue warrants more elaborate description, also update the [Upgrade Guide](../website/content/docs/upgrade/upgrade-specific.mdx).
+
+Use the PR number as the file name. Enterprise private changes should have an entry in the OSS repository referencing a public GitHub issues detailing the change and impact.
+
+Below are some examples of how to generate a CHANGELOG entry with your pull
+request.
+
+### Improvement
+
+~~~
+```release-note:improvement
+internal/server: Add new option for configs
```

Comment:
Double checking - these lines shouldn't include PR/Issue #'s & links? (which will be derived from the filename automatically instead?)

Reply:
Correct. PR link is automatic.